tests/containers/kubectl.pm

```diff
@@ -102,6 +102,8 @@ sub run {
     validate_script_output('kubectl describe deployments/nginx-deployment | grep Replicas', sub { $_ =~ m/5 desired.*5 total/ });
     # Scale in
     assert_script_run('kubectl scale --replicas=2 deployments/nginx-deployment');
+    # When scaling healthy cluster in the deployment condition is always available even when more than specified amount of replicas still exist
+    validate_script_output('kubectl get events | grep ScalingReplicaSet | tail -n1', sub { $_ =~ m/Scaled down replica set nginx-deployment-.*to 2/ });
     assert_script_run('kubectl wait deployment nginx-deployment --for condition=available --timeout=300s', timeout => 330);
     validate_script_output('kubectl describe deployments/nginx-deployment | grep Replicas', sub { $_ =~ m/2 desired.*2 total/ });
```

Comment:
I think line 108 "kubectl desctibe | grep Replicas" still is incorrect and could generate the same error again. Yes we'll have 2 desired, but maybe not 2 in total because is a scaling down and "kubectl wait" is not doing what we think here because is not blocking until the pods are destroyed.

Reply:
@ilausuch please check now.

Reply:
Thank you for your explanation (in a meeting) :)